index.html

```diff
@@ -151,22 +480,211 @@
                 </nav>
                 <ul class="header_actions">
                     <li>
+<<<<<<< HEAD
+=======
+
+>>>>>>> f0b2564849cd39b199e8404147467499001595c7
+                        <button class="action_item"><img src="Project materials\search .icn-xs.svg"
+                                alt="Search"></button>
+                    </li>
+                    <li>
+<<<<<<< HEAD
+                        <button class="action_item"><img src="Project materials\cart .icn-xs.svg" alt="Cart"></button>
+=======
+                        <button class="action_item"><img src="Project materials\cart .icn-xs.svg" alt="Cart"> </button>
                         <button class="action_item"><img src="Project materials\search .icn-xs.svg" alt="Search"> </button>
                     </li>
                     <li>
                         <button class="action_item"><img src="Project materials\cart .icn-xs.svg" alt="Cart" > </button>
+
+>>>>>>> f0b2564849cd39b199e8404147467499001595c7
                     </li>
                 </ul>
             </header>
             <div class="top_banner_content">
                 <h1 id="future">We Take Care of the Future</h1>
-                <h4 id="weknow">We know how large objects will act, but things on a small scale just do not act that way</h4>
+                <h4 id="weknow">We know how large objects will act, but things on a small scale just do not act that way
+                </h4>
                 <div class="top_banner_actions">
                     <button class="bttn bttn--primary">Get Quote Now</button>
-                    <button class="bttn bttn--secondary">Learn More</button>
```

Comment:
I guess here should be bttn--secondary instead of bttn--learn. 
Check please if the current class is the one you wanted to use